test/test_scanner.py

```diff
@@ -235,6 +235,11 @@ def condensed_filepath(self, url, package_name):
                 d["version"],
                 d["other_products"],
                 marks=[
+                    # if other_products are not mentioned then this test will be skipped.
+                    pytest.mark.skipif(
+                        "other_products" is None,
+                        reason=f"other_products are not mentioned.",
+                    ),
```

Comment:
@singh-anushka I don't think skipping the entire test is the right way to go about this issue as we want an other_products=[] to be automatically considered for checkers having no such parameter mentioned. However we do not want to skip the test completely for such cases.

Reply:
This is actually for older checkers; because they do not have this parameter mentioned. So we can skip the tests. But the can modify the docs accordingly So that in future something people would keep on adding the "other_products" parameter.
This was my logic behind this statement.